lib/utils/isStandardSyntaxRule.js

```diff
@@ -22,36 +22,13 @@ module.exports = function (rule) {
 		return false;
 	}
 
-	// Called Less mixin (e.g. a { .mixin() })
-	// @ts-expect-error -- TODO TYPES support LESS and SASS types somehow
-	if (rule.mixin) {
-		return false;
-	}
-
-	// Less detached rulesets
-	if (selector.startsWith('@') && selector.endsWith(':')) {
-		return false;
-	}
-
-	// Ignore Less &:extend rule
-	if ('extend' in rule && rule.extend) {
-		return false;
-	}
```

Comment:
[suggestion] This check of the `exptend` property seems effective still to me because `postcss-less` add the property. See below:

```console
$ node -pe 'require("postcss-less").parse("&:extend(.inline) {}").nodes[0]'
<ref *1> Rule {
  raws: { before: '', between: ' ', after: '' },
  type: 'rule',
  nodes: [],
  parent: Root {
    raws: { semicolon: false, after: '' },
    type: 'root',
    nodes: [ [Circular *1] ],
    source: { input: [Input], start: [Object] },
    [Symbol(isClean)]: false
  },
  source: {
    start: { offset: 0, line: 1, column: 1 },
    input: Input {
      css: '&:extend(.inline) {}',
      hasBOM: false,
      id: '<input css qtn4s3>',
      [Symbol(fromOffset cache)]: [Array]
    },
    end: { offset: 19, line: 1, column: 20 }
  },
  selector: '&:extend(.inline)',
  extend: true,
  [Symbol(isClean)]: false
}
```

So, how about moving this `extend` check before calling `isStandardSyntaxSelector()` as below?

```js
	if (rule.type !== 'rule') {
		return false;
	}

	// Ignore Less &:extend rule
	if ('extend' in rule && rule.extend) {
		return false;
	}
 ```

Reply:
Done.
Thank you!